Src/WitsmlExplorer.Api/Workers/CreateRiskWorker.cs

```diff
@@ -0,0 +1,120 @@
+using System;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Serilog;
+using Witsml;
+using Witsml.Data;
+using Witsml.Extensions;
+using Witsml.Query;
```

Comment:
Is it possible to move the `Witsml.Query` namespace into `WitsmlExplorer.Api.Query` so that we build further on the refactoring done after issue #350